github-bot/harvester_github_bot/config.py

```diff
@@ -1,25 +1,80 @@
+from flask import Flask
 from everett.component import RequiredConfigMixin, ConfigOptions
 from everett.manager import ConfigManager, ConfigOSEnv
+from werkzeug.security import generate_password_hash
+from github import Github
+
+from harvester_github_bot.zenhub import Zenhub
+
+import logging
+
+FLASK_LOGLEVEL = ""
+FLASK_PASSWORD = ""
+FLASK_USERNAME = ""
+GITHUB_OWNER = ""
+GITHUB_REPOSITORY = ""
+GITHUB_REPOSITORY_TEST = ""
+ZENHUB_PIPELINE = ""
+BACKPORT_LABEL_KEY = ""
+
+app = Flask(__name__)
+gapi = {}
+zapi = {}
```

Comment:
maybe defining it as `zenh_api` and `gh_api` that is more obvious to the readers.